.github/workflows/pr_docs.yaml

```diff
@@ -0,0 +1,34 @@
+name: pr_docs
+
+on:
+  push:
+    paths:
+      - "docs/**.md"
+      - "**.md"
+  pull_request:
+    paths-ignore:
+      - "docs/**.md"
+      - "**.md"
+    branches:
+      - main
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT || secrets.GITHUB_TOKEN }}
+          
+      - name: MegaLinter documentation flavor
+        uses: oxsecurity/megalinter/flavors/documentation@v6.12.0
+        env:
+          IGNORE_GITIGNORED_FILES: true
+          VALIDATE_ALL_CODEBASE: true
+          PRINT_ALL_FILES: true
+          DISABLE: SPELL,COPYPASTE,YAML
```

Comment:
I feel we should keep spell-check ON. What are your and team's thoughts on this?

Reply:
I think it is good to keep it on for documentation linting. I will change it.